### PR TITLE
Add flycheck-erlang-include-path variable

### PR DESCRIPTION
**Pull request by yzprofile**

### CODE REVIEW

File: flycheck.el [hunk 1]
```diff
@@ -5886,11 +5886,25 @@ The checker runs `checkdoc-current-buffer'."
   (setcar (get 'emacs-lisp-checkdoc 'flycheck-command)
           flycheck-this-emacs-executable))
 
+(flycheck-def-option-var flycheck-erlang-include-path nil erlang
+  "Whether to error on language extensions in Erlang.
```
Comment (lunaryorn) [inline flycheck.el]: Please fix this docsring.

File: flycheck.el [hunk 2]
```diff
@@ -5886,11 +5886,25 @@ The checker runs `checkdoc-current-buffer'."
...
+
+The value of this variable is a list of strings, where each
+string is a directory to add to the include path of erlang.
```
Comment (lunaryorn) [inline flycheck.el]: `… to the include path of `erlc'`, please.  That's clearer.

**Comment by lunaryorn** [comment]
Please add this option to the manual, at `docs/languages.texi`.

**Comment by yzprofile** [comment]
Is there anything need to be modified?

**Comment by lunaryorn** [comment]
@yzprofile No, but Github does not notify about changes to a PR, hence I overlooked your fixes.  Merged.

Thanks for your contribution!

**Pull request merged**